### PR TITLE
Bootstrap app after forking

### PR DESCRIPTION
**Pull request by CloudNiner**
## Overview

Updates:
- Dependencies in package.json
- Dependencies in bower.json
- Deployments to both a staging and production environment
- Instructions for how to create an environment in AWS along with `./scripts/infra` for regular deployment

## Testing

Follow setup instructions in "Development" section of README. Ensure you're able to run the dev site and it looks like MuseumStat. 

Proof of deployment is that the following sites exist, use HTTPS and work:
- http://staging.impactview.org
- http://impactview.org

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -55,8 +55,8 @@
 
     <!-- build:js(.) scripts/vendor.js -->
     <!-- bower:js -->
-    <script src="bower_components/jquery/dist/jquery.js"></script>
     <script src="bower_components/angular/angular.js"></script>
+    <script src="bower_components/jquery/dist/jquery.js"></script>
```
Comment (KlaasH) [inline app/index.html]: Was having jQuery first causing something to fail?  This way means AngularJS will be using its own jqLite for `$element`, right?  Which is fine, except in those occasional instances where one wants to do something unusual with `$element` and having the full jQuery comes in handy.

File: app/scripts/acs/acs-graphs.js [hunk 2]
```diff
@@ -61,27 +61,27 @@
...
-                                    ' (', percent, '%)'].join('');
-                        })
                         .staggerLabels(true)
                         .color(function () { return '#bc5405'; })
                         .margin(opts.margin);
+                    chart.tooltip.contentGenerator(function (d) {
+                        var percent = numberFilter(d.data.value / total * 100, 1);
+                        return ['<b>', d.data.label, ': </b>', numberFilter(d.data.value),
+                                ' (', percent, '%)'].join('');
+                    });
```
Comment (KlaasH) [inline app/scripts/acs/acs-graphs.js]: These are identical, right?  Moving them right below the conditional would work the same and only require one copy.

File: app/scripts/acs/acs-graphs.js [hunk 3]
```diff
@@ -106,14 +106,14 @@
...
-                                ' (', percent, '%)'].join('');
-                    })
                     .color(function () { return '#bc5405'; })
                     .margin({top: 0, left: 10, bottom: 0, right: 10})
                     .showLegend(false);
+                chart.tooltip.contentGenerator(function (d) {
+                    var percent = $filter('number')(d.data.value / total * 100, 1);
+                    return ['<b>', d.data.label, ': </b>', numberFilter(d.data.value),
+                            ' (', percent, '%)'].join('');
+                });
```
Comment (KlaasH) [inline app/scripts/acs/acs-graphs.js]: Here it is again.  Now it seems like a helper function is in order (moving the one above outside the conditional still makes sense, even though it'll be shorter).

File: .travis.yml [hunk 4]
```diff
@@ -6,9 +6,7 @@ cache:
         - bower_components
 
 node_js:
-  - '0.12'
-  - '4.2'
-  - '5.1'
+  - '6'
```
Comment (KlaasH) [inline .travis.yml:9]: The README still says "nodejs 0.12+".  Should that be changed?

**Review by KlaasH — APPROVED** [review]
A few notes and questions below, but it provisioned (after I installed ruby-dev and ruby 2.0--I was on 1.9.1, which `sass` didn't like) and came up fine.

:+1:

File: app/index.html (hunk 1 above)
Comment (CloudNiner) [reply app/index.html]: Hmm. Good eye. I didn't manually change this, the order was modified by the package updates I did. I'll see about reverting it.

**Pull request merged**